velero-2-mixin/links.libsonnet

```diff
@@ -4,7 +4,10 @@ local g = import './g.libsonnet';
   new(this):
     {
       veleroClusterOverview:
-        link.link.new('Velero cluster overview', '/d/' + this.grafana.dashboards.clusterOverview.uid)
+        link.link.new('Velero cluster view', '/d/' + this.grafana.dashboards.clusterOverview.uid)
+        + link.link.options.withKeepTime(true),
+      veleroOverview:
```

Comment:
It looks like the links are not working on the cloud. The cluster dashboard has the old name, and the logs dashboard link isn't showing up.

Reply:
This isn't a problem, just something that happened due to the way i progressed through the mixin and branches. This will be fixed when i merge everything to the base mixin